### ISSUE TITLE
Avatar.Image should take standard Image props

### ISSUE THREAD

**Issue by suitupalex** [callstack/react-native-paper#1066]
## Describe the feature

Standard Image props like onError, onLoad, accessibilityLabel, etc. should be able through to the Avatar.Image component.

## Motivation

One example is, I'd like to fallback to Avatar.Text if loading an image via URI fails.

Off-topic question: I tried reimplementing Avatar.Image myself since it's so simple, but whenever I use my own component in a FlatList, it terribly lags after about 60 items. When I use Paper's Avatar.Image, everything renders nice and speedily. Is there anything different Paper is doing under the hood to optimize performance? I don't see anything obvious.

**Comment by satya164** [comment]
> Standard Image props like onError, onLoad, accessibilityLabel, etc. should be able through to the Avatar.Image component.

PR welcome

> Is there anything different Paper is doing under the hood to optimize performance

Nope

**Comment by suitupalex** [comment]
Would love to make a PR. Should the props be declared in the component or can they be passed through with a spread operator? I'd imagine the latter to be future proofed, right?

**Comment by satya164** [comment]
Spreading rest of the props is fine I think

**Comment by tiendn** [comment]
Spreading rest of the props is fine, but I think define exactly props is better.
With the support of IDE, we don't need to read Official API which props have defined.

**Comment by aprilmintacpineda** [comment]
So I guess this didn't happen?
